openedx/core/djangoapps/user_api/helpers.py

```diff
@@ -141,9 +144,9 @@ def __init__(self, method, submit_url):
         self._field_overrides = defaultdict(dict)
 
     def add_field(
-        self, name, label=u"", field_type=u"text", default=u"",
-        placeholder=u"", instructions=u"", required=True, restrictions=None,
-        options=None, include_default_option=False, error_messages=None
+            self, name, label=u"", field_type=u"text", default=u"",
+            placeholder=u"", instructions=u"", required=True, restrictions=None,
+            options=None, include_default_option=False, error_messages=None,
```

Comment:
Is this the preferred indentation for long argument specs in edx code? I seem to recall seeing the previous indentation more in the platform.